### ISSUE TITLE
Deprecation warning about the use of ast.Bytes

### ISSUE THREAD

**Issue by fm2003** [flake8-implicit-str-concat/flake8-implicit-str-concat#67]
I'm getting this warning:

`.../user_code/.local/share/virtualenvs/user_code-fKLzWnXE/lib/python3.12/site-packages/flake8_implicit_str_concat.py:58: DeprecationWarning: ast.Bytes is deprecated and will be removed in Python 3.14; use ast.Constant instead`

And also this:

`DeprecationWarning: ast.Str is deprecated and will be removed in Python 3.14; use ast.Constant instead`

Both are referring to line 58 in `flake8_implicit_str_concat.py`: 

```py
isinstance(operand, (ast.Str, ast.Bytes, ast.JoinedStr))
```

**Comment by hugovk** [comment]
Step 1: add unit tests, which reveal the deprecation warnings: https://github.com/flake8-implicit-str-concat/flake8-implicit-str-concat/pull/69
